src/Symfony/Component/Routing/Matcher/Dumper/DumperCollection.php

```diff
@@ -47,8 +47,6 @@ public function all()
 
     /**
      * Adds a route or collection.
-     *
-     * @param DumperRoute|DumperCollection The route or collection
```

Comment:
Cannot be removed, I think it was removed as it's not valid. `$child` is missing here.

Reply:
Yes, this is better to fix manually, since this Fixer relies on valid code.